dashboard/app/helpers/aichat_ai_client.rb

```diff
@@ -26,12 +26,14 @@ def get_response_text(config, request, context = [])
 
     body = create_body(config, request, context)
 
+    read_timeout = DCDO.get('openai_http_read_timeout', SharedConstants::AI_CHAT_READ_TIMEOUTS[config[:clientType]] || 30)
```

Comment:
Not necessary here, but I wonder if we should remove DCDO here as the layers of overrides/defaults feels a bit confusing.

Reply:
It seems it might be useful to continue supporting DCDO in case a situation arises in which we want to increase timeouts quickly without a deploy?  Certainly, it will override the special-casing for the longer timeouts, but perhaps still useful for our dominant use cases?